docs/api/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>JoinMarket Wallet RPC API Reference</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style>
+    body { margin: 0; padding: 0; }
+  </style>
+</head>
+<body>
+  <redoc spec-url="./wallet-rpc.yaml"></redoc>
+  <script src="https://cdn.jsdelivr.net/npm/redoc@latest/bundles/redoc.standalone.min.js"></script>
```

Comment:
Post merge comment - I really don't like this! Can't we host that JS locally?

Reply:
Yes, good point, will send in a fix! Just checking if there are more things to be addressed after this merge.

Reply:
Addressed in #1154.